src/main/java/jenkins/plugins/git/GitToolChooser.java

```diff
@@ -59,7 +60,10 @@ public GitToolChooser(@NonNull AbstractGitSCMSource source) throws IOException,
      * Estimate size of a repository using the extension point
      * @param remoteName: The URL of the repository
      */
-    public GitToolChooser(String remoteName) {
+    public GitToolChooser(String remoteName, Boolean useJGit) {
+        if (useJGit != null) {
+            JGIT_SUPPORTED = useJGit;
+        }
```

Comment:
My intention is to create instantiate GitToolChooser with both repo url and JGit compatibility info, but the info should be an optional parameter.

Agreed, `JGIT_SUPPORTED` should be set `false` by default instead of `true`.

Reply:
> Agreed, JGIT_SUPPORTED should be set false by default instead of true.

It's still set to `true` by default.

Reply:
Apologies for missing it earlier, I have added it in the latest commit.